src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <header>
+      <form
+        class="search-form"
+        aria-label="search-form"
+        method="get"
+        action="/"
+        data-qa="big"
+      >
+        <div class="search-bar search-bar__content">
+          <input
```

Comment:
It is better to use label instead of div

Reply:
Good afternoon! Thanks 
Already fixed.